app/src/main/res/raw/server_info.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Could you put this under a `/chicago/server_info.json` or `/tampa/server_info.json` folder so we can easily tell the source? That will make it easier to add more cities in the future too. I think we're missing tests for the Tampa data right now in OTP? If so could you please add them?